### PR TITLE
Rename models and operations with mixed cases

### PR DESCRIPTION
**Pull request by therve**
A few operations and models are affected by a small problem where a
character gets isolated. This fixes it.

### CODE REVIEW

File: .generator/src/generator/openapi.py [hunk 1]
```diff
@@ -286,6 +286,11 @@ def operation(spec, operation_id):
     return None
 
 
+def safe_snake_case(value):
+    value = value.replace("IdP", "Idp").replace("AuthN", "Authn").replace("IoT", "Iot")
```
Comment (jirikuncar) [inline .generator/src/generator/openapi.py:290]: can we have these keywords somewhere as a global mapping?

**Review by jirikuncar — APPROVED** [review]

File: tests/conftest.py [hunk 2]
```diff
@@ -65,6 +65,7 @@ def pytest_terminal_summary(terminalreporter, exitstatus, config):
 
 logging.basicConfig()
 
+EDGE_CASES = {"IdP": "Idp", "AuthNMapping": "AuthnMapping", "AuthN ": "Authn ", "IoT": "Iot"}
```
Comment (jirikuncar) [inline tests/conftest.py:68]: could we have it as a json file that is shared with generator?

Comment (therve) [reply tests/conftest.py:68]: Do you have a preferred location? Probably somewhere in .generator right?

Comment (jirikuncar) [reply tests/conftest.py:68]: yes .generator seems appropriate

**Pull request merged**